### PR TITLE
87 initialize new blank request

### PR DESCRIPTION
**Pull request by alishaevn**
# story
 I made this branch based off #114. after it gets merged, I can merge that code into here so we see what the real differences here are. otherwise, this code is good for review
- able to render a request for a ware without a dynamic form
- able to get the input data

# demo
image

### CODE REVIEW

File: pages/requests/new/[ware].js [hunk 1]
```diff
@@ -69,14 +71,15 @@ const NewRequest = () => {
   }
 
   const handleSubmit = (event) => {
-    // NOTE(alishaevn): leaving this commented out code because it may come back into play with the blank form
-    // event.preventDefault()
-    // event.stopPropagation()
-    // setValidated(true)
+    if (!event.formData) {
```
Comment (alishaevn) [inline pages/requests/new/[ware].js:74]: `event.formData` is only a thing on the react json schema form.

File: pages/requests/new/[ware].js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import React, { useState } from 'react'
+import { default as BsForm } from 'react-bootstrap/Form'
```
Comment (alishaevn) [inline pages/requests/new/[ware].js:2]: we need this for when there isn't a dynamic form on the ware.

File: pages/requests/new/[ware].js [hunk 3]
```diff
@@ -97,28 +100,52 @@ const NewRequest = () => {
...
+            requestForm={requestForm}
+            updateRequestForm={updateRequestForm}
+          />
+        </BsForm>
+      )}
     </div>
   )
 }
 
+const StandardRequestOptions = ({ defaultRequiredDate, requestForm, updateRequestForm, }) => (
```
Comment (alishaevn) [inline pages/requests/new/[ware].js:128]: these request options are the "shipping/billing info" and "additional info" components that are needed on the page whether or not we have a dynamic form. I moved them into a component to DRY the code a bit.

File: pages/requests/new/[ware].js [hunk 4]
```diff
@@ -97,28 +100,52 @@ const NewRequest = () => {
...
         </Form>
-      }
+      ) : (
+        <BsForm
+          onSubmit={handleSubmit}
+          id={`new-${id}-request-form`}
+          noValidate
+          validated={validated}
+        >
+          <BlankRequestForm updateRequestForm={updateRequestForm} />
```
Comment (alishaevn) [inline pages/requests/new/[ware].js:116]: since we're rendering the blank request form here, we don't need the "pages/requests/new/index.js" file.

**Review by K8Sewell — APPROVED** [review]

**Pull request merged**